cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -496,6 +502,18 @@ fromConsensusQueryResult QuerySystemStart q' r' =
         -> r'
       _ -> fromConsensusQueryResultMismatch
 
+fromConsensusQueryResult QueryChainBlockNo q' r' =
+    case q' of
+      Consensus.GetChainBlockNo
+        -> r'
+      _ -> fromConsensusQueryResultMismatch
+
+fromConsensusQueryResult (QueryChainPoint mode) q' r' =
```

Comment:
The reason I need the `mode` is because below, `fromConsensusPointInMode` requires `mode`.